mmcv/runner/fp16_utils.py

```diff
@@ -122,8 +121,8 @@ def new_func(*args, **kwargs):
                     else:
                         new_kwargs[arg_name] = arg_value
             # apply converted arguments to the decorated method
-            if (TORCH_VERSION != 'parrots'
-                    and LooseVersion(TORCH_VERSION) >= LooseVersion('1.6.0')):
+            if (TORCH_VERSION != 'parrots' and
+                    digit_version(TORCH_VERSION) >= digit_version('1.6.0')):
```

Comment:
should digit_version deal with "parrots"?

Reply:
We do not deal with "parrots" in version comparison, "parrots" is dealt like a special case.

Reply:
Notice this pattern suggests that the digit_version should not accept "parrots" as input. If this is the case, then we can add an assertion to enforce

![image](https://user-images.githubusercontent.com/9464825/126580366-70c01ebe-bb93-40bd-a012-ec5e48f6d438.png)

Reply:
OK, I will add an assertion in digit_version to make sure that.